installers/cubeit-installer

```diff
@@ -132,6 +133,7 @@ if [ -z "${CONFIG_FILES}" ]; then
     CONFIG_FILES="config-installer.sh"
 fi
 
+source ${DEFAULT_CONFIG}
```

Comment:
This makes no sense. The installer is already capable of sourcing the configs that are passed when it is run. They are tuned for the install task at hand .. we do not want any default configs to be sourced.